swf/src/types.rs

```diff
@@ -1786,6 +1786,7 @@ pub type DoAction<'a> = &'a [u8];
 pub type JpegTables<'a> = &'a [u8];
 
 /// `ProductInfo` contains information about the software used to generate the SWF.
```

Comment:
nit: Repeating the type name is redundant here, just like it was for `enum_trait_object`.

Reply:
fixed